winml/api/Windows.AI.MachineLearning.idl

```diff
@@ -213,6 +213,15 @@ namespace ROOT_NS.AI.MachineLearning
         //! BatchSizeOverride > 0 indicates the size of batch that will be used to override the model batch size and optimize evaluations.
         UInt32 BatchSizeOverride { get; set; };
 
+        [contract(MachineLearningContract, 4)]
```

Comment:
please note that you will need to make corresponding changes in the OS to have this take effect inbox:
https://microsoft.visualstudio.com/OS/_git/os.2020?path=%2Fonecoreuap%2Fmerged%2FSDK%2FSDKPlatforms%2FUAP%2FPlatform.xml&version=GBofficial%2Frsmaster&_a=contents